packages/agent/src/server/loader.ts

```
@@ -0,0 +1,225 @@
+      const character = await jsonToCharacter(responseJson);
+      characters.push(character);
+    }
+    return characters;
+  } catch (e) {
+    logger.error(`Error loading character(s) from ${url}: ${e}`);
+    process.exit(1);
+  }
+}
+
+export async function jsonToCharacter(
+  character: any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)